src/selection.ts

```diff
@@ -7,15 +7,15 @@ import {ParameterName} from './parameter';
 import {Dict} from './util';
 
 export const SELECTION_ID = '_vgsid_';
-export type SelectionType = 'point' | 'interval';
+export type SelectionType = 'point' | 'interval' | 'lasso';
```

Comment:
For some context, when first designing selections, we intentionally chose `interval` to emphasize _what_ was being selected (rather `brush` which would have focused on _how_ that selection was occurring). 

I think a similar analogy applies here. Rather than `lasso` (which, to me, describes _how_ the selection is performed), I wonder if we can find names that better describe _what_ is being selected?

Off the top of my head, the closest candidate I can think of is `region`.